packages/myst-cli/src/process/site.ts

```diff
@@ -283,9 +287,7 @@ export async function processProject(
       // Load all citations (.bib)
       ...project.bibliography.map((path) => loadFile(session, path, siteProject.path, '.bib')),
       // Load all content (.md and .ipynb)
-      ...pages.map((page) =>
-        loadFile(session, page.file, siteProject.path, undefined, { minifyMaxCharacters }),
-      ),
+      ...pages.map((page) => loadFile(session, page.file, siteProject.path, undefined)),
```

Comment:
I am still a bit concerned that we are no longer passing this option in at all? I think this actually might be breaking JATS - @fwkoch I would love another set of eyes here.